globals.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+/**
+ * @typedef {{x: number, y: number}}    Point
```

Comment:
Do you know why this is showing up as a new file?

Reply:
The location changed. This is in root instead of in `src/`. Dunno why, though

Reply:
I uploaded it in wrong location